Makefile

```diff
@@ -4,6 +4,8 @@ VERSION ?= SNAPSHOT-${GIT_COMMIT_SHORT}
 LDFLAGS_VERSION=-X github.com/astronomer/astro-cli/version.CurrVersion=${VERSION}
 ENVTEST_ASSETS_DIR=$(shell pwd)/testbin
 
+CORE_OPENAPI_SPEC=../astro/apps/core/docs/public/public_v1alpha1.yaml
```

Comment:
could we add GitHub URL instead? It might not be necessary for everyone to have that file locally, and more importantly, in case someone has an older version locally, we might risk overwriting it with stale code.

Reply:
astro is a private repo so this file can not be read publicly.  any idea how to do it in an easy way?

Reply:
I don't think there might be a straightforward way to do it without exposing tokens, since this repo is public. So let's leave it as it is